lib/modules/datasource/rubygems/metadata-cache.ts

```diff
@@ -63,7 +62,6 @@ export class MetadataCache {
       );
       return data;
     } catch (err) {
-      logger.debug({ err }, 'Rubygems: failed to fetch metadata');
```

Comment:
I think people will take this log message too seriously and report it as a bug.
We can do nothing about servers that don't support endpoints with metadata.

Reply:
You could keep the debug message, and improve it to explain why it's not a big problem. You could also lower the log level from `err` to something appropiate.